src/test/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminFacilityServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.dts.fact.services.admin.list;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.entity.FacilityType;
+import uk.gov.hmcts.dts.fact.repositories.FacilityTypeRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith({SpringExtension.class, MockitoExtension.class})
```

Comment:
Don't think you need to extend with the MockitoExtention here because you are not using the @Mock annotation in this class. In your test below, you are constructing a list of facilities and not a mock

Reply:
Tried removing the MockitoExtention but test wouldn't run so left annotation on